test/test.articles.js

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs';
+import 'dotenv/config';
+import Helper from '../src/helpers/helper';
+import { chai, server, expect } from './test-setup';
+
+const usertoken = Helper.generateToken({
+  id: 2,
+  email: 'user@gmail.com',
+  username: 'user',
+  verified: false
+});
+
+describe('Articles', () => {
+  it('throw error when fields are missing', (done) => {
+    chai
+      .request(server)
+      .post('/api/v1/articles')
+      .set('Content-Type', 'application/json')
+      .set('Authorization', usertoken)
+      .field('tagList', 'tag1, tag2, tag3')
+      .attach('images', fs.readFileSync(`${__dirname}/mock/img.png`), 'img.png')
+      .end((err, res) => {
+        expect(res.status).to.be.deep.equal(400);
+        expect(res.body).to.have.deep.property('status');
+      });
+    done();
+  });
+  it('create an article', (done) => {
+    chai
+      .request(server)
+      .post('/api/v1/articles')
+      .set('Content-Type', 'application/json')
+      .set('Authorization', usertoken)
+      .field('title', 'Title')
+      .field('body', 'body field')
+      .field('description', 'description is here')
+      .field('tagList', 'tag1, tag2, tag3')
+      .attach('images', fs.readFileSync(`${__dirname}/mock/img.png`), 'img.png')
```

Comment:
Mock the attachment of images on cloudinary

Reply:
@Mnickii, I managed to mock upload images and the test is passing both locally and on travis-ci; the problem is on signup which is failing on travis-ci.

Reply:
@Mnickii now test is mocked.